src/components/Header.js

```diff
@@ -19,9 +19,8 @@ class Header extends React.Component {
 
     render() {
         return (
-            <>
                 <div>
-                    {this.ok}
+                    {this.ok} 
```

Comment:
What is it supposed to to ?

Reply:
oh sorry ! It was a test i've did before to do my responsive navbar and i forgot to delete it !